README.md

```diff
@@ -65,7 +70,7 @@ Convert the following create method in Mongoose to ActiveRecord.
 How does module.exports help us with separation of concerns?
 
 ```text
-
+Module.exports by returning an object as a result of a require call which cause each function to do one thing at a time.
```

Comment:
The first half looks great; "Module.exports by returning an object as a result of a require call".  I'm not sure what you mean by "...which cause each function to do one thing at a time."